packages/puppeteer-core/src/common/Binding.ts

```diff
@@ -8,17 +8,27 @@ import {debugError} from './util.js';
  * @internal
  */
 export class Binding {
+  #identifier: string;
```

Comment:
I am not sure if we should add it here as it is really the identifier for the evaluated script, right?

Reply:
That is correct. An other option is having a Map with `exposedFunction.name` to `identifier` on the Page, so similar.

Reply:
what do you think if we change the exportFunction to return some sort of object identifying the function?